### DISCUSSION TITLE
Orphaned Blocks

### DISCUSSION THREAD

**Discussion by VahanN95** [smartcontractkit/full-blockchain-solidity-course-py#1142] [Q&A] [answered]
Hello, 

I am pretty new in this topic and have a question. 
Lets say we have have 3 miners `M1 M2 M3`. And 5 transactions `T1 T2 T3 T4 T5`.
And lets assume that 
```
M1 -> T1, T2, T3
M2  -> T2, T4
M3 -> T1, T4, T5

```

And M1 mined successfully and added new block to the chain.  But on the other hand M2 and M3 mined their blocks as well.
So the question is what will happen with other transactions of Orphaned blocks and how they will be retried?
And Second question. After M1 broadcasted its block `T1, T2, T3` will exists in the block chain. How the system will make sure that these transactions will not be reused?

**Comment by cromewar** [comment, marked as answer]
Hello @VahanN95 
> what will happen with other transactions of Orphaned blocks and how they will be retried


This is due to the confirmations on the blockchain, let me go in deep into it.

1. Each node has a copy of the blockchain, so the nodes are  competing with each other to be the ones to register the transaction... when a node wins that competition the rest of the nodes works as confirmations, making sure the block the node is trying to register is the same as them, at the end the Sakamoto consensus uses the largest chain which has 51% coincidence as reference
2. 
> How the system will make sure that these transactions will not be reused?
2. After a block is written there are more confirmation, which are block confirmations, basically means more blocks are added after the block on where your transaction was registered, if there is more blocks it means that the blockchain  is correct and the transactions were permanently added as immutable of the blockchain, meaning the cannot be reused.

**Reply by VahanN95** [reply]
So dose that means that per every block all the transactions are checked if it is existing or not? 
So in the example I have mentioned M2 and M3 have some transactions which will be in the blockchain but some other are not. So Ho the miners are understanding what transactions are pending?

**Reply by cromewar** [reply]
This happens automatically using math, basically the first to solve the problem to register the transaction is the one who proposes that register/block, if other nodes gets to the same result and confirm the transaction is legit it the transaction/block get written permanently. The don't "know" if the others are pending or not, they just try to make the register, but if the register was already proposed they are used just as confirmations.

**Reply by VahanN95** [reply]
Thank yo so much!
